tests/cookiecutter.go

```diff
@@ -0,0 +1,38 @@
+package tests
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+type Choices string
+
+const (
+	Yes Choices = "1"
+	No  Choices = "2"
```

Comment:
Just curious, why do you use "1", "2" instead of "yes"/"no"?

Reply:
Because when you use choices variable in `cookiecutter.json`, Cookiecutter will ask you to put in the number instead of value.

<img width="524" alt="Screen Shot 2564-07-02 at 17 06 29" src="https://user-images.githubusercontent.com/29707647/124439798-0ef14500-dda4-11eb-8efa-f79e32a8aad4.png">